helm_deploy/values-dev.yaml

```diff
@@ -28,6 +28,9 @@ generic-service:
     API_BASE_URL_PERSONAL_RELATIONSHIPS: https://personal-relationships-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_ORGANISATIONS: https://organisations-api-dev.hmpps.service.justice.gov.uk
     FEATURE_EVENT_PERSONALRELATIONSHIPS_OFFENDER_PHYSICAL_DETAILS-CHANGED: false
+    FEATURE_EVENT_OFFENDER_SENTENCES-INSERTED: false
+    FEATURE_EVENT_OFFENDER_SENTENCES-DELETED: false
+    FEATURE_EVENT_OFFENDER_SENTENCES-UPDATED: false
 
```

Comment:
these don't look right - think I'm missing the queue/organisation. Will revisit in next commit, the events aren't being emitted yet

Reply:
They are fine; you only need additional "domain" key if there is likely to be a conflict with another project; e.g somebody else was interested in OFFENDER_SENTENCES-*